### PR TITLE
WindowsMaps HideInfoWindow

### PR DESCRIPTION
**Pull request by VladislavAntonyuk**
### Description of Change ###

HideInfoWindow if args is set to true

Methods SendMarkerClick/SendInfoWindowClick return a boolean parameter, indicating if the info window should be closed.

 ### PR Checklist ###
 
 - [ ] Has a linked Issue, and the Issue has been `approved`(bug) or `Championed` (feature/proposal)
 - [ ] Has tests (if omitted, state reason in description)
 - [ ] Has samples (if omitted, state reason in description)
 - [ ] Rebased on top of `main` at time of PR
 - [ ] Changes adhere to [coding standard](https://github.com/CommunityToolkit/Maui/blob/main/CONTRIBUTING.md#contributing-code---best-practices)
 - [ ] Documentation created or updated: https://github.com/MicrosoftDocs/CommunityToolkit/pull/XYZ 

 ### Additional information ###

### CODE REVIEW

File: src/CommunityToolkit.Maui.Maps/Handler/Map/MapHandler.Windows.cs [hunk 1]
```diff
@@ -445,8 +461,16 @@ void WebViewWebMessageReceived(WebView2 sender, CoreWebView2WebMessageReceivedEv
...
+					if (clickedPin is null)
+					{
+						break;
+					}
+
+					var hideInfoWindow = clickedPin.SendMarkerClick();
+					if (hideInfoWindow)
+					{
+						CallJSMethod(PlatformView, "hideInfoWindow();");
+					}
```
Comment (pictos) [inline src/CommunityToolkit.Maui.Maps/Handler/Map/MapHandler.Windows.cs:473]: This code is the same as on line `444`, can we refactor this to a method, passing the `clickedPin` and just call it?

Comment (VladislavAntonyuk) [reply src/CommunityToolkit.Maui.Maps/Handler/Map/MapHandler.Windows.cs:473]: it is not the same. in one case we call `clickedPin.SendMarkerClick()`, in another case `clickedPin.SendInfoWindowClick()`

Comment (pictos) [reply src/CommunityToolkit.Maui.Maps/Handler/Map/MapHandler.Windows.cs]: Oh my bad then...

File: src/CommunityToolkit.Maui.Maps/Handler/Map/MapHandler.Windows.cs [hunk 2]
```diff
@@ -433,8 +440,12 @@ void WebViewWebMessageReceived(WebView2 sender, CoreWebView2WebMessageReceivedEv
 				if (!string.IsNullOrEmpty(clickedInfoWindowWebViewId))
 				{
 					var clickedPin = VirtualView.Pins.SingleOrDefault(p => (p as Pin)?.Id.ToString().Equals(clickedInfoWindowWebViewId) ?? false);
-
-					clickedPin?.SendInfoWindowClick();
+					
+					var hideInfoWindow = clickedPin?.SendInfoWindowClick();
+					if (hideInfoWindow is not false)
```
Comment (JoonghyunCho) [inline src/CommunityToolkit.Maui.Maps/Handler/Map/MapHandler.Windows.cs:445]: why not just `is true`?

**Review by JoonghyunCho — APPROVED** [review]
Looks good to me

**Pull request merged**